moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -188,11 +191,10 @@ struct FCLGeometry
   }
 
   std::shared_ptr<fcl::CollisionGeometry> collision_geometry_;
-  boost::shared_ptr<CollisionGeometryData>  collision_geometry_data_;
+  CollisionGeometryDataPtr collision_geometry_data_;
 };
 
-typedef std::shared_ptr<FCLGeometry> FCLGeometryPtr;
-typedef std::shared_ptr<const FCLGeometry> FCLGeometryConstPtr;
+MOVEIT_CLASS_FORWARD(FCLGeometry);
```

Comment:
Yeah I noticed. `FCLGeometry` here is not actually a type from FCL, so there was no need to make it `std::shared_ptr`. While I'm all for using `std::shared_ptr` over the boost one, I figured consistency is better and switch this one at the same time as the others. But I can revert this bit of course.